speedtest/datadog_checks/speedtest/data/conf.yaml.example

```diff
@@ -0,0 +1,54 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## There are no options here
+
+## Every instance is scheduled independent of the others.
+#
+instances:
+
+  -
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check emitted by this instance.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param host - string - optional
```

Comment:
new config options need to be added to `speedtest/assets/configuration/spec.yaml`. This file is generated with `ddev --extras validate config --sync speedtest`

Reply:
there is literally 0 documentation on this: https://a.cl.ly/P8uee58x

Reply:
Hi! Our dev docs are available here: https://datadoghq.dev/integrations-core/meta/config-specs/#templates

I don't believe they've been indexed by the internal docs

Reply:
I guessed my way through this, but seems right now